packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -137,6 +154,34 @@ const MenuButton = styled.button`
     display: none;
     visibility: hidden;
   }
+  @media (min-width: ${GEL_GROUP_B_MIN_WIDTH}rem) {
+    ${({ script }) => {
+      const sideLength = `${calculateButtonSide(
+        script.pica.groupB.lineHeight,
+      )}rem;`;
+      return (
+        script &&
+        `height: ${sideLength};
+        width: ${sideLength};`
+      );
+    }}
+  }
+  @media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}rem) {
```

Comment:
Wondering if this should be `max-width`, as the menuButton will be displayed below 600px

Reply:
This makes sure the height of the menu button is the correct height if it ever is visible in `GEL_GROUP_CD`. If we think it's not worth having that "just in case" (and it probably isn't as long as if this button is ever visible at this breakpoint we put this back in) then this whole media query can just be removed.

Let me know what you think whether it's worth just removing this.

Reply:
Ah, ok. I'm for removing it for now as it feels a bit premature to add it before it is actually required. Curious what others think though

Reply:
Probably best to remove it for now as it is not being used anyway

Reply:
![](https://media.giphy.com/media/kqu0JnieiuhGw/giphy-downsized.gif)